types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@joystream/types",
-  "version": "0.18.5",
+  "version": "0.18.19",
```

Comment:
I think that should be `0.19.0` instead?

Reply:
fixed in 0c5da6f04d68bfc5f7fd85be110d4353de42680f